rbd-target-gw.py

```diff
@@ -300,7 +300,7 @@ def define_luns(gateway):
                         try:
                             with rbd.Image(ioctx, image_name) as rbd_image:
                                 image_bytes = rbd_image.size()
-                                image_size_h = human_size(image_bytes)
+                                image_size_h = str(image_bytes) + 'b'
 
                                 lun = LUN(logger, pool, image_name,
```

Comment:
Nit: why not just pass the ``image_bytes``` to ```LUN``` and let that class convert it to human-readable. Right now, we convert the bytes to "human readable" and then ```LUN``` re-converts it back to bytes.

Reply:
Do we need compat support for outside users? If not then I think what you suggest is best. I think we just need to at least convert the ceph-iscsi-cli/ansible code then.

Reply:
@mikechristie I think if can be parsed as an int (no suffix), we can assume it's in bytes.

Reply:
Ah ok. Sounds nice so internally we do not have to do the extra step.

Reply:
@dillaman 

Did you mean LUN() would accept an int or string? If so is the python'y way to do this:

LUN(.... size)

try:
   size_bytes = int(size)
except ValueError:
   size_bytes convert_2_bytes(size)

Or did you just mean it would always be a string and if there was no suffix then it would be assumed to be in bytes.

Reply:
The former — try it as an int and if it works, it’s in bytes.